.nvmrc

```diff
@@ -1 +1 @@
-12.18.3
+14.16.0
```

Comment:
That's the Node version Electron@12 is using (see https://www.electronjs.org/releases/stable#12.0.0)